src/main/java/de/tum/in/www1/artemis/web/rest/admin/AdminStandardizedCompetencyResource.java

```diff
@@ -25,10 +26,13 @@
 import de.tum.in.www1.artemis.service.competency.StandardizedCompetencyService;
 import de.tum.in.www1.artemis.web.rest.dto.competency.KnowledgeAreaDTO;
 import de.tum.in.www1.artemis.web.rest.dto.competency.StandardizedCompetencyDTO;
+import de.tum.in.www1.artemis.web.rest.dto.standardizedCompetency.KnowledgeAreaResultDTO;
+import de.tum.in.www1.artemis.web.rest.dto.standardizedCompetency.StandardizedCompetencyResultDTO;
 
 /**
  * Admin REST controller for managing {@link StandardizedCompetency} entities.
  */
+@Validated
```

Comment:
What exactly do you use this for?

According to the web: https://stackoverflow.com/questions/36173332/difference-between-valid-and-validated-in-spring

We don't utilise any validation groups here.